sdk/ml/azure-ai-ml/CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 ### Other Changes
  - Removed declaration on Python 3.6 support
  - Added support for custom setup scripts on compute instances.
+ - Removed declaration on Python 3.6 support.
```

Comment:
This is a duplicate of line 16

Reply:
Fixed, also simplified the PR since some changed files were deleted.